packages/shared-ux/button_toolbar/src/buttons/add_from_library/add_from_library.tsx

```diff
@@ -8,9 +8,9 @@
 
 import { i18n } from '@kbn/i18n';
 import React from 'react';
-import { PrimaryButton, Props as ToolbarButtonProps } from '../primary/primary';
+import { ToolbarButton, ToolbarButtonProps } from '../toolbar_button';
 
-export type Props = Omit<ToolbarButtonProps, 'iconType' | 'label'>;
+export type Props = Omit<ToolbarButtonProps, 'iconType' | 'label' | 'type'>;
```

Comment:
I made the assumption that the `AddFromLibrary` button should never be a primary button - it's only ever a secondary action in current solutions (Dashboard and Canvas). If this is the wrong assumption, we could allow the `AddFromLibrary` button to take the `type` property, which we could pass down to the `ToolbarButton` component.